README.md

```diff
@@ -904,27 +928,154 @@ class TritonPythonModel:
       for infer_response in infer_responses:
         # Check if the inference response has an error
         if inference_response.has_error():
-            raise pb_utils.TritonModelException(inference_response.error().message())
+            raise pb_utils.TritonModelException(
+              inference_response.error().message())
         else:
             # Extract the output tensors from the inference response.
-            output1 = pb_utils.get_output_tensor_by_name(inference_response, 'REQUESTED_OUTPUT_1')
-            output2 = pb_utils.get_output_tensor_by_name(inference_response, 'REQUESTED_OUTPUT_2')
+            output1 = pb_utils.get_output_tensor_by_name(
+              inference_response, 'REQUESTED_OUTPUT_1')
+            output2 = pb_utils.get_output_tensor_by_name(
+              inference_response, 'REQUESTED_OUTPUT_2')
 
-            # Decide the next steps for model execution based on the received output
-            # tensors.
+            # Decide the next steps for model execution based on the received
+            # output tensors.
 ```
 
 A complete example for sync and async BLS in Python backend is included in the
 [Examples](#examples) section.
 
+Starting from 23.02 release, you can execute inference requests on decoupled
+models in both [default mode](#default-mode) and
+[decoupled mode](#decoupled-mode). By setting the parameter `decoupled` to
```

Comment:
"By setting the parameter `decoupled` to `True`" -> "By setting the `decoupled parameter to `True`,